tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -43,11 +43,14 @@ C_CONTROLMIXER(Master Playback Volume, PIPELINE_ID,
 # Volume configuration
```

Comment:
I forgot to mention the top-level topologies impacted. I'll add that to next version commit message. I'll first check that draft version CI is all green.

Reply:
Done.